res/values/strings.xml

```diff
@@ -1734,12 +1721,12 @@
     <string name="registration_activity__the_registration_lock_pin_is_not_the_same_as_the_sms_verification_code_you_just_received_please_enter_the_pin_you_previously_configured_in_the_application">The Registration Lock PIN is not the same as the SMS verification code you just received. Please enter the PIN you previously configured in the application.</string>
     <string name="registration_activity__registration_lock_pin">Registration Lock PIN</string>
     <string name="registration_activity__forgot_pin">Forgot PIN?</string>
-    <string name="registration_lock_dialog_view__the_pin_can_consist_of_four_or_more_digits_if_you_forget_your_pin_you_could_be_locked_out_of_your_account_for_up_to_seven_days">The PIN can consist of four or more digits. If you forget your PIN, you could be locked out of your account for up to seven days.</string>
+    <string name="registration_lock_dialog_view__the_pin_can_consist_of_four_or_more_digits_if_you_forget_your_pin_you_could_be_locked_out_of_your_account_for_up_to_seven_days">Make up your own PIN, which must consist of four or more digits. If you forget your PIN, you could be locked out of your Signal account for up to seven days.</string>
     <string name="registration_lock_dialog_view__enter_pin">Enter PIN</string>
     <string name="registration_lock_dialog_view__confirm_pin">Confirm PIN</string>
     <string name="registration_lock_reminder_view__enter_your_registration_lock_pin">Enter your Registration Lock PIN</string>
     <string name="registration_lock_reminder_view__enter_pin">Enter PIN</string>
-    <string name="preferences_app_protection__enable_a_registration_lock_pin_that_will_be_required">Enable a Registration Lock PIN that will be required to register this phone number with Signal again.</string>
+    <string name="preferences_app_protection__enable_a_registration_lock_pin_that_will_be_required">Configure a PIN which will from then on be required to register this phone number with Signal again. By doing this you prevent someone who hijacked your phone number from also using Signal in your name.</string>
```

Comment:
with your proposed change:
- I wonder who is requiring
- I am missing the signal words "from then on"

Not only does this prevent impersonation, but also receiving of messages in your name, which is why I haven't used the word impersonation.

Reply:
Why are you excluding "receiving of messages in your name" (or the original "using Signal in your name") as a act of impersonation?

Reply:
using signal includes receiving? I will think about this more

Reply:
I don't fully understand your question @valldrac  . I think if we use the word "impersonating" it will not be entirely clear to the user what that implies in practice. What it implies in practice is that the person would be abel to send and receive messages and calls in your name. I shortned "send and receive messages and calls" to "use Signal".

Reply:
I cannot assess right now if the meaning of "impersonating" is clear for the average user. So go ahead with this for now.